analytics/src/analytics/metrics/percent_complete.py

```diff
@@ -110,23 +102,24 @@ def _get_count_by_deliverable(
         # group by deliverable and sum the values in the unit field
         # then rename the sum column to the value of the status var
         # to prevent duplicate col names when open and closed counts are joined
-        df_agg = df.groupby(self.deliverable_col, as_index=False).agg({unit: "sum"})
-        return df_agg.rename(columns={unit: status})
+        df_agg = df.groupby(self.deliverable_col, as_index=False).agg({unit_col: "sum"})
+        return df_agg.rename(columns={unit_col: status})
 
     def _prepare_result_dataframe_for_plotly(self) -> pd.DataFrame:
         """Stack the open and closed counts self.results for plotly charts."""
         # unpivot open and closed counts so that each deliverable has both
         # an open and a closed row with just one column for count
+        unit = self.unit.value
```

Comment:
tiny nitpick: personally i find it a little confusing to name this `unit` when we already have, in other places (i.e. in `self.unit`), a variable called `unit` where the corresponding value that's equal to this is `unit.value`. So in one place, `unit` has a property `value`, and that `value` is called `unit` in another place. 

Consider renaming this version of `unit` to something like `unit_value`?